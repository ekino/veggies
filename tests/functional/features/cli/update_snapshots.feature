@updateSnapshots
Feature: Dummy test for --updateSnapshots

    @long
    Scenario: Should echo salut1
        When I run command echo salut1
        Then exit code should be 0
        And stdout output should match snapshot

    @short
    Scenario: Should echo salut2
        When I run command echo salut2
        Then exit code should be 0
        And stdout output should match snapshot
