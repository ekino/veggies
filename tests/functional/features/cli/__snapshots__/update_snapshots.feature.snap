

exports[`Should echo salut1 1.1`] = `"salut1
"`;

exports[`Should echo salut2 1.1`] = `"salut2
"`;
